Userland/Libraries/LibCore/IOSurface.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2024, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/OwnPtr.h>
+#include <LibCore/IOSurface.h>
+
+#if !defined(AK_OS_MACOS)
+static_assert(false, "This file must only be used for macOS");
+#endif
+
+#define FixedPoint FixedPointMacOS
+#define Duration DurationMacOS
+#include <IOSurface/IOSurface.h>
+#undef FixedPoint
+#undef Duration
+
+namespace Core {
+
+#define kIOSurfacePixelFormatBGRA8888 'BGRA'
```

Comment:
This is strange. Is that not a define from Apple already? either way, using a define for this feels not very C++-brained, we should at the very least use a constexpr variable.

Also, as `'BGRA', this becomes a multi-byte character literal. Which also looks odd.

Reply:
it does not seem to be defined in Apple headers, but I removed `define`.